### PR TITLE
[Debian package] Move launcher to /usr/bin

### PR DESCRIPTION
**Pull request by Glandos**
Starts to work on #497.
The launcher is moved from `/usr/share/jitsi-videobridge/jvb.sh` into `/usr/bin/jvb`.
This is *far* from perfect, since it only works for linux-64, but I want to gather comments before going on.
If the way to do it seems wrong, please raise your hand.

(Sorry for other commits, I've rebased too late)

### CODE REVIEW

File: resources/install/build-debian.xml [hunk 1]
```diff
@@ -123,6 +121,16 @@
...
+        <chown owner="root">
+            <fileset dir="${debian.dir}/usr/bin"></fileset>
+        </chown>
+        <chgrp group="root">
+            <fileset dir="${debian.dir}/usr/bin"></fileset>
+        </chgrp>
+        <chmod perm="755">
+            <fileset dir="${debian.dir}/usr/bin"></fileset>
+        </chmod>
+
```
Comment (damencho) [inline resources/install/build-debian.xml]: Why you need to touch some system folder permissions which are not jvb related.

File: resources/install/linux-64/jvb.sh [hunk 2]
```diff
@@ -15,13 +15,13 @@ if [[ "$1" == "--help"  || $# -lt 1 ]]; then
...
 
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+BASE_LIB_DIR="$(echo ~jvb)"
 
 mainClass="org.jitsi.videobridge.Main"
-cp=$(JARS=($SCRIPT_DIR/jitsi-videobridge*.jar $SCRIPT_DIR/lib/*.jar); IFS=:; echo "${JARS[*]}")
-libs="$SCRIPT_DIR/lib/native/linux-64"
-logging_config="$SCRIPT_DIR/lib/logging.properties"
-videobridge_rc="$SCRIPT_DIR/lib/videobridge.rc"
+cp="$BASE_LIB_DIR/*:$BASE_LIB_DIR/lib/*"
```
Comment (damencho) [inline resources/install/linux-64/jvb.sh]: I think cp was containing a list of jars and not star paths, not sure whether the later works.

File: resources/install/build-debian.xml (hunk 1 above)
Comment (Glandos) [inline resources/install/build-debian.xml]: You're right, I just did this for no reason.

File: resources/install/linux-64/jvb.sh (hunk 2 above)
Comment (Glandos) [inline resources/install/linux-64/jvb.sh]: See http://docs.oracle.com/javase/8/docs/technotes/tools/unix/classpath.html#A1100762
and specifically:

> Class path entries can contain the base name wildcard character (*), which is considered equivalent to specifying a list of all of the files in the directory with the extension .jar or .JAR

Comment (damencho) [inline resources/install/linux-64/jvb.sh]: I think I've been experiencing this problem and I think there is a reason that current -cp behave like this. Have you tested this?

**Comment by bgrozev** [comment]
This is now obsolete, please re-open if it's still needed.

**Pull request closed without merging**